docs/user-guide/slicing.ipynb

```diff
@@ -133,6 +133,25 @@
     "- Slicing **with a range**:\n",
     "  - The *coordinates* for the sliced dimension are *kept*.\n",
     "\n",
+    "The rationale behind this mechanism is as follows.\n",
+    "We may want to modify slices independently, e.g., by adding an offset to certin slices:"
+   ]
+  },
+  {
```

Comment:
You could show what the slice `DatasetView` contains that might help reinforce the rules above.

Reply:
Added reference to figure below.